packages/components/src/utils/validation/throwIfElementIsNotOfKind.spec.ts

```diff
@@ -0,0 +1,37 @@
+import * as getPrefixedTagNamesUtils from '../tag-name';
+import { throwIfElementIsNotOfKind } from './throwIfElementIsNotOfKind';
+
+const host = document.createElement('div');
+const link = document.createElement('p-link') as unknown as HTMLSlotElement;
+
+it('should call getPrefixedTagNames() with correct parameters', () => {
+  const spy = jest.spyOn(getPrefixedTagNamesUtils, 'getPrefixedTagNames');
+
+  throwIfElementIsNotOfKind(host, link, 'p-link');
+
+  expect(spy).toBeCalledWith(host);
+});
+
+it('should call getTagName() with correct parameters', () => {
+  const spy = jest.spyOn(getPrefixedTagNamesUtils, 'getTagName');
+
+  throwIfElementIsNotOfKind(host, link, 'p-link');
+
+  expect(spy).toBeCalledWith(link);
+});
+
+it('should throw error if slot is not of kind', () => {
```

Comment:
Relationship between the utils that are verified being called in the previous tests and the thrown exception is missing. 
They could just be called without an assignment or into a console.log.